packages/desktop-client/src/components/transactions/SimpleTransactionsTable.jsx

```diff
@@ -155,7 +153,12 @@ export function SimpleTransactionsTable({
   const memoFields = useMemo(() => fields, [JSON.stringify(fields)]);
 
   const serializedTransactions = useMemo(() => {
-    return transactions.map(trans => serializeTransaction(trans, dateFormat));
+    transactions = transactions.map(trans =>
+      serializeTransaction(trans, dateFormat),
+    );
+    const selected = transactions.filter(x => x.selected).map(x => x.id);
+    selected.forEach(x => dispatchSelected({ type: 'select', id: x }));
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏actions (`dispatch`) should not be called in memoization. Instead: use `useEffect` if you need to react to the data changing.

Reply:
Adjusted, 
&& Learned what memoization is...